examples/experimental/i3s-picking/tile-layer/tile-layer.js

```diff
@@ -52,12 +52,22 @@ export default class TileLayer extends Tile3DLayer {
 
     const {attributeStorageInfo} = tile.tileset.tileset;
     const {layerFeaturesAttributes} = tile.header.userData;
-
     const featureAttributes = {};
+    let calculatedFeatureIndex = null;
 
     for (let index = 0; index < attributeStorageInfo.length; index++) {
       const attributeName = attributeStorageInfo[index].name;
-      const attributeValue = layerFeaturesAttributes[index][attributeName][featureIndex];
+      const attributeData = layerFeaturesAttributes[index][attributeName];
```

Comment:
- Maybe state clearly in comment that this logic is for 1.6? 
- Maybe add a helper function that calculates it? 
- Since it doesn't reference `this` it could be a global helper function. rather than a member.
- Could this logic be moved into the I3SLoader?
- This logic should also should have a test case.